src/charts/createDotPlot.js

```diff
@@ -39,26 +39,26 @@ export function createDotPlot(this_, d) {
   const chartData = d.statistics.values
     .sort(
       (a, b) =>
-        (a.prop_n > b.prop_n
+        a.prop_n > b.prop_n
           ? -2
-          : a.prop_n < b.prop_n ? 2 : a.key < b.key ? -1 : 1)
+          : a.prop_n < b.prop_n ? 2 : a.key < b.key ? -1 : 1
```

Comment:
could be! Maybe we should explicitly set the version in package.json? I rarely run `npm update` on these since dependencies are pretty stable.